account_payment_order/models/account_move_line.py

```diff
@@ -6,49 +6,82 @@
 
 from odoo import api, fields, models
 from odoo.fields import first
-from odoo.osv import orm
+
+from odoo.addons.base.models.ir_ui_view import (
+    transfer_field_to_modifiers,
+    transfer_modifiers_to_node,
+    transfer_node_to_modifiers,
+)
+
+
+def setup_modifiers(node, field=None, context=None, in_tree_view=False):
```

Comment:
Why do you need this? I think the previous algorithm was not very good and check also the problem depicted in #683. The goal of this is to hide some fields in a specific view, so maybe there can be better options (like a primary view).

Reply:
Solved with a primary view!